metabase/metabase
# CSV / Admin Upload Settings

Resolves https://github.com/metabase/metabase/issues/29601

### Description

Adds settings to enable uploads and customize where they go.

Depends on backend in https://github.com/metabase/metabase/pull/29681

![Screen Shot 2023-03-30 at 5 04 02 PM](https://user-images.githubusercontent.com/30528226/228983657-0d778d9e-70cb-4204-aa1e-f42372394651.png)

### How to verify

1. Go to admin settings > uploads
2. You should be able to enable uploads
3. you should be able to select only actions-enabled databases
4. you should be able to designate and upload schema or table prefix

> **Note**
> We should make schema only selectable on databases that support schema, like postgres. That proved to be pretty tricky within our settings framework. Will follow up on that detail later.


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/29709)
